src/main/java/com/minecolonies/coremod/proxy/CommonProxy.java

```diff
@@ -25,13 +29,24 @@
 import java.util.HashMap;
 import java.util.Map;
 
-
 /**
  * CommonProxy of the minecolonies mod (Server and Client).
  */
 public class CommonProxy implements IProxy
 {
     /**
+     * Constants used for Registering Entities and Eggs
+     */
+    private static final int TRACKING_RANGE            = 256;
```

Comment:
As stated these are constants in the common Proxy (so used on both sides), move them to the constant class please.